### ISSUE TITLE
AI section

### ISSUE THREAD

**Issue by piamancini** [DemocracyEarth/paper#13]
I feel the whole AI section is distracting from the main goal of the paper. The references to YC are too niche, US centric, SV biased and irrelevant to the global democracy audience. I can't understand what the added value is of being so silicon valley and the detriments (exponentially widens the attack space of these ideas) are too big. 

If a point must be made, I suggest it's worked as a paragraph in the section of the cloud vs the land.

**Comment by santisiri** [comment]
I think that we are building a solid case on AI as it is a mainstream topic that even though it might be perceived US centric, it does really concern the world in the same way Facebook has a global impact. I'm hesitant to remove our arguing here, AI is a critical part of governance.

**Comment by dwightmwilson** [comment]
I agree w both Santi re the importance and Pia about the length so I might suggest cutting the last 2 paragraphs of this suggestion.  I imagine, Santi, that you really like the last paragraph (as do I) but the big statements in there are soooo big that they almost need a few more pages of explication, which we don't have.  And "consciousness" as the new political frontier is very cool but also leaves the reader hanging: "what does it mean?"  "i thought DEF is about the very practical businesses of software and incorruptible voting and governing the critical issues of our lives, but "consciousness" etc

**Comment by santisiri** [comment]
consciousness is politics. im touching on a core aspect of ai with it.

**Comment by SFSandra** [comment]
I took a stab at shortening the AI section while retaining the core aspect of consciousness - most recent pull request. If you don't like, just ignore, but I've managed to cut several hundred words. 

 I've  I've retained the excised material, we can write a companion/follow up piece to publish in the Hacktivism Medium pub.....

**Comment by bhack** [comment]
If you are not limited just to a voting platform there is also this very big topic to handle https://github.com/conversationai/perspectiveapi/issues/2

**Comment by santisiri** [comment]
@bhack you seem to be discussing that with @iislucas, a friend of mine from jigsaw. definitely a very relevant topic but we tried to focus strictly on voting for the purpose of this paper. do you work in jigsaw as well?

**Comment by bhack** [comment]
No I am not in jigsaw. But I think that focusing only on voting it is a big limit cause without a scalability of the discussion and topics the main problem still be unsolved: who decides what to vote.
So the gate keepers will be still in a dominant position. 
This was also the main unsolved problem of DAO or other direct/liquid democracy efforts. 
Also a voting proposal is still a topic if it needs to catch consensus/quorum to be voted. Human attention is limited and popularity could overcome minority position in the opinion formation process.
If you don't want to solve this problem with DemocracyEarth I think that we still need to talk about this in the paper for "intellectual integrity".

**Comment by santisiri** [comment]
@bhack paper is open source, so feel free to make contributions as you see fit extending or modifying sections. regarding the attention issue, we address that very much in detail across Section 3. i agree that discussion/deliberation matters at lot, we chose to focus on voting but i think that many of the conversationalai ideas could be a good fit for the 'agora' section in the paper.

**Comment by bhack** [comment]
A good summary of [open ethics problems](https://alexgkendall.com/artificial_intelligence/lets_talk_about_ethics_in_artificial_intelligence/)

**Comment by santisiri** [comment]
that's a great reference, thanks

**Comment by bhack** [comment]
If you are interested in this topic you can find a good update resource at http://mattchessen.com/ai/the-artificial-intelligence-policy-landscape/

**Comment by iislucas** [comment]
And our breakdown of thinking about bias in ML is here: https://conversationai.github.io/bias.html
(Nice to see this work happening! keep it up!)

**Comment by santisiri** [comment]
thanks @iislucas :) very happy to see that our paths keep crossing

**Comment by bhack** [comment]
https://youtu.be/59bMh59JQDo
